src/library/dom/convertCssWidthPropertyToPixels.ts

```diff
@@ -48,7 +48,7 @@ export const convertCssWidthPropertyValueToPixels = ({
       const parent = element.parentElement;
       if (!parent) {
         throw new Error(
-          "The element uses a percent unit, but is unattached to a parent to calculate a percent from."
+          "The element uses a percent unit, but is unattached to a parent to calculate a percent from.",
```

Comment:
Of course, none of the added commas are for arrays, but worth the price I think :). Am I misguided?